specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -3371,13 +3511,13 @@
       "description": "Defines the parameters for RequestBody match conditions",
       "required": [
         "operator",
-        "@odata.type"
+        "typeName"
       ],
       "properties": {
-        "@odata.type": {
+        "typeName": {
           "type": "string",
           "enum": [
-            "#Microsoft.Azure.Cdn.Models.DeliveryRuleRequestBodyConditionParameters"
+            "DeliveryRuleRequestBodyConditionParameters"
```

Comment:
Any reason for this kind of breaking changes?

Also it is advised to add `x-ms-enum` to all enums.

Reply:
We removed this value since we are no longer supporting OData type.